MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2546,10 +2546,16 @@ protected string GetHardwareAcceleratedVideoDecoder(EncodingJobInfo state, Encod
                 return null;
             }
 
-            if (videoStream != null
-                && !string.IsNullOrEmpty(videoStream.Codec)
-                && !string.IsNullOrEmpty(encodingOptions.HardwareAccelerationType))
+            if (IsCopyCodec(state.OutputVideoCodec))
```

Comment:
```suggestion
            if (EncodingHelper.IsCopyCodec(state.OutputVideoCodec))
```

Reply:
Why? IsCopyDoc is a member of EncodingHelper it seems strange to me to use the full name. In fact it was my IDE that suggested that the "Name can be simplified"

Reply:
For code consistency.
But now I prefer to change the remaining four `EncodingHelper.IsCopyCodec` like yours.

Reply:
Makes sense. I'll do the change